packages/devtools_app/lib/src/screens/inspector/inspector_service.dart

```diff
@@ -838,21 +838,21 @@ abstract class ObjectGroupBase implements Disposable {
     );
   }
 
-  List<RemoteDiagnosticsNode?> parseDiagnosticsNodesHelper(
+  List<RemoteDiagnosticsNode> parseDiagnosticsNodesHelper(
     List<Object>? jsonObject,
     RemoteDiagnosticsNode? parent,
     bool isProperty,
   ) {
     if (disposed || jsonObject == null) return const [];
-    final List<RemoteDiagnosticsNode?> nodes = [];
+    final List<RemoteDiagnosticsNode> nodes = [];
```

Comment:
type on RHS

Reply:
done

Reply:
looks like this type is still on the left - move to the RHS

Reply:
now for sure :)